[FIX] stock_currency_valuation: Fixed compute standard_price_in_currency

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/stock#839

feg-adhoc committed:
[FIX] stock_currency_valuation: Fixed compute standard_price_in_currency

X-original-commit: 355cedeb7e37d0ab0592120ef2b34bc0d485bb15

roboadhoc commented:
[![Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/stock/pull/840.png)](https://mergebot.adhoc.ar/ingadhoc/stock/pull/840)

fw-bot-adhoc commented:
@feg-adhoc @maq-adhoc cherrypicking of pull request ingadhoc/stock#839 failed.

stdout:
```
CONFLICT (modify/delete): stock_currency_valuation/__manifest__.py deleted in 3784c0c5e8b4bd898f799177edc2cb1d84e28ca9 and modified in a0263ec124ea9b0b623ef5d570e2bd5e6fe70ed4.  Version a0263ec124ea9b0b623ef5d570e2bd5e6fe70ed4 of stock_currency_valuation/__manifest__.py left in tree.
CONFLICT (modify/delete): stock_currency_valuation/models/product_product.py deleted in 3784c0c5e8b4bd898f799177edc2cb1d84e28ca9 and modified in a0263ec124ea9b0b623ef5d570e2bd5e6fe70ed4.  Version a0263ec124ea9b0b623ef5d570e2bd5e6fe70ed4 of stock_currency_valuation/models/product_product.py left in tree.
CONFLICT (modify/delete): stock_currency_valuation/models/product_template.py deleted in 3784c0c5e8b4bd898f799177edc2cb1d84e28ca9 and modified in a0263ec124ea9b0b623ef5d570e2bd5e6fe70ed4.  Version a0263ec124ea9b0b623ef5d570e2bd5e6fe70ed4 of stock_currency_valuation/models/product_template.py left in tree.
CONFLICT (modify/delete): stock_currency_valuation/views/product.xml deleted in 3784c0c5e8b4bd898f799177edc2cb1d84e28ca9 and modified in a0263ec124ea9b0b623ef5d570e2bd5e6fe70ed4.  Version a0263ec124ea9b0b623ef5d570e2bd5e6fe70ed4 of stock_currency_valuation/views/product.xml left in tree.

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

:warning: after resolving this conflict, you will need to merge it via @roboadhoc.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port

fw-bot-adhoc commented:
@feg-adhoc @maq-adhoc this forward port of ingadhoc/stock#839 is awaiting action (not merged or closed).
